manifest.uuid

```diff
@@ -0,0 +1 @@
+4552f9fc70c62ea7dd443604b73f9cacf0aa1c4b
```

Comment:
What's that?

Reply:
@stgraber see comment about manifest file above.